components/Chips/src/Theming/MDCChipView+MaterialTheming.m

```diff
@@ -17,6 +17,7 @@
 #import "MaterialChips+ColorThemer.h"
 #import "MaterialChips+ShapeThemer.h"
 #import "MaterialChips+TypographyThemer.h"
```

Comment:
Can't we remove this import as well? If so, should we remove the BUILD target and CocoaPod?

Reply:
Done.